pkgs/development/tools/database/pgsync/Gemfile.lock

```diff
@@ -2,19 +2,19 @@ GEM
   remote: https://rubygems.org/
   specs:
     parallel (1.22.1)
-    pg (1.4.4)
-    pgsync (0.7.2)
+    pg (1.4.5)
+    pgsync (0.7.3)
       parallel
       pg (>= 0.18.2)
       slop (>= 4.8.2)
       tty-spinner
-    slop (4.9.3)
+    slop (4.10.0)
     tty-cursor (0.7.1)
     tty-spinner (0.9.3)
       tty-cursor (~> 0.7)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
This should be `ruby`, otherwise it won't be usable on other platforms
```suggestion
  ruby
```

Reply:
Thanks for the review, amended the mistaken change of platform! :+1: